core/src/SYCL/Kokkos_SYCL.cpp

```diff
@@ -143,119 +148,117 @@ void SYCL::impl_initialize(SYCL::SYCLDevice d) {
   Impl::SYCLInternal::singleton().initialize(d.get_device());
 }
 
-std::ostream& SYCL::SYCLDevice::info(std::ostream& os) const {
+std::ostream& SYCL::info(std::ostream& os, const sycl::device& device) {
```

Comment:
How about naming this impl_sycl_info. I am not insisting on it, but it took me a while to recognize this is a private function and thus its fine to have it as something which isn't part of the execution space concept. Since its private I am not against it, we may actually jsut want to discuss in the next meeting naming practices for private functions - if we want one or not.